firefox-ios/Client/Frontend/Browser/Toolbars/Models/AddressToolbarContainerModel.swift

```diff
@@ -29,7 +29,13 @@ class AddressToolbarContainerModel {
             searchEngineImage: profile.searchEngines.defaultEngine?.image,
             lockIconImageName: StandardImageIdentifiers.Medium.lock,
             url: url,
-            searchTerm: searchTermFromURL(url, searchEngines: profile.searchEngines))
+            searchTerm: searchTermFromURL(url, searchEngines: profile.searchEngines)) {
```

Comment:
I think it would be better to use the parameter name (`onTapLockIcon`) here instead of the trailing closure as it makes it clearer what the closure is for.